apps/cowswap-frontend/src/modules/swap/containers/SwapConfirmModal/index.tsx

```
@@ -0,0 +1,100 @@
+import { useRateInfoParams } from 'common/hooks/useRateInfoParams'
+import { CurrencyPreviewInfo } from 'common/pure/CurrencyAmountPreview'
+
+import { useLabelsAndTooltips } from './useLabelsAndTooltips'
+
+import { useSwapDerivedState } from '../../hooks/useSwapDerivedState'
+
+const CONFIRM_TITLE = 'Swap'
+const PRICE_UPDATE_INTERVAL = ms`30s`
+
+export interface SwapConfirmModalProps {
+  doTrade(): Promise<false | void>
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)